mate-panel/panel-menu-items.h

```diff
@@ -26,6 +26,7 @@
 
 #include <gtk/gtk.h>
 #include "panel-widget.h"
+#include <libmate-desktop/mate-image-menu-item.h>
```

Comment:
This is just a dummy comment to prevent merging before acknowledging https://github.com/mate-desktop/mate-desktop/pull/603 is merged and the build succeeds.

The only change that might be required would be adding a dependency on the newer mate-desktop (with the above merged), but it is apparently controversial… I don't want to block anything though, so I'll stop at having explained my concerns about not doing so :)

Reply:
I also have no intention of merging this prior to the mate-desktop commit upon which it depends. I do not know however what to do about the 1.28 release issue. This could be put off by branching 1.28 now, bumping version in master to 1.29, and if folks want to use this in 1.28 we can deal with it then. If we do that, we can leave 1.28 alone, knowing we have code that works only by chance, but this does bet on nothing changing on the GTK side. Fortunately, a deprecated widget in GTK3 is now unlikely to get a change in it's size. Leaving that question for the others

Reply:
I know for the merging, but my approval made it a little too easy to merge this in the UI by accident, hence adding an unresolved discussion that blocks merging :)

IMO we should branch 1.28 anyway so we are ready to work on the dev version as usual while also being able to do 1.28 stuff.  But this could happen after this is merged as well if we want it in

Reply:
What should I do at this point?